doc/objectives.md

```diff
@@ -0,0 +1,18 @@
+# Study Objectives
+
+- **Is there a relationship between predator size (mass, length, fork-length) and prey size (mass, length, fork-length)?**
+  
+  There will be a trend of larger predators consuming larger prey. 
+  
+- **Does the same species of predator in different geographic locations exhibit the same feeding pattern? Are they eating the same sizes of prey in different locations?**
+  
+  Locations with more similar environmental factors will have predators exhibiting similar feeding preferences. 
+  
+- **Does geography (latitude, longitude, geographic location, specific habitat) influence predator and prey sizes (mass or length ratios)?**
+
+  Different locations will exhibit different patterns in predator and prey size. 
+
```

Comment:
And then here it's more of us predicting that predator and prey size vary across space (i don't know if we wanted this to be about the relationship between them predator and prey, or just hypothesizing that the mass/length ratio varies spatially in general). What do you guys think?

Reply:
I think it would be interesting if it were the about predator-prey relationship. The first step would probably be to see how the mass/length ratio varies spatially, but then we could build off that and look into the predator-prey relationship